website/docs/r/key_vault_certificate.html.markdown

```diff
@@ -251,6 +251,9 @@ The `certificate` block supports the following:
 * `contents` - (Required) The base64-encoded certificate contents.
 * `password` - (Optional) The password associated with the certificate.
 
+~> **NOTE:** A PEM certificate is already base64 encoded. To successfully import contents should include a PEM encoded X509 certificate and a private_key in pkcs8 format (use 
+`openssl pkcs8 -topk8 -nocrypt -in private_key.pem > private_key_pk8.pem`). There should only be linux style \n line endings and the whole block should have the PEM begin/end blocks around the certificate data and the private key data. These terminators should be generated when PEM documents are created and look like `-----BEGIN CERTIFICATE-----`,`-----END CERTIFICATE-----` and `-----BEGIN PRIVATE KEY-----`, `-----END PRIVATE KEY-----`.
```

Comment:
I think the example terminators might be better illustrated in a code block rather than inline? Perhaps even with dummy cert/key